README.md

```diff
@@ -60,6 +60,19 @@ RuboCop::RakeTask.new do |task|
 end
 ```
 
+### Rubocop rules for RBI files
+
+Put this into your `.rubocop.yml`:
+
+```yaml
+inherit_gem:
+  rubocop-sorbet: config/rbi.yml
+
+AllCops:
+  Include:
+    - '**/*.rbi'
```

Comment:
Can't we have this in `rbi.yml` so that it only applies to `**/*.rbi` files by default?